rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -8,11 +8,20 @@ local chash = require("balancer.chash")
 local chashsubset = require("balancer.chashsubset")
 local sticky_balanced = require("balancer.sticky_balanced")
 local sticky_persistent = require("balancer.sticky_persistent")
-local ewma = require("balancer.ewma")
+local ewma      = require("balancer.ewma")
+local string    = string
+local ipairs    = ipairs
+local table     = table
+local getmetatable = getmetatable
+local tostring  = tostring
+local pairs     = pairs
+local math      = math
+
```

Comment:
I don't like aligning things like this unless there's something like gofmt that would do it for me.

Reply:
ok, fixed already